src/constants/ember-organizations/ember-a11y.js

```diff
@@ -0,0 +1,26 @@
+const GithubOrganization = require('../../classes/github-organization');
+
+const githubOrganization = new GithubOrganization({
+  organizationName: 'ember-a11y',
+
+  repositoryNames: [
+    'a11y-announcer',
+    'a11y-demo-app',
+    'core-notes',
+    'ember-a11y-landmarks',
+    'ember-a11y-refocus',
+    'ember-a11y-testing',
+    'ember-a11y.com',
+    'ember-a11y',
+    'ember-component-focus',
```

Comment:
`ember-a11y` had an `issues` repo, but it seemed like an internal repo not meant for open source contribution (commented out in case).

```suggestion
    'ember-a11y',
    'ember-a11y-landmarks',
    'ember-a11y-refocus',
    'ember-a11y-testing',
    'ember-a11y.com',
    'ember-component-focus',
    // 'issues',
```